doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.postprocess.ConsolidateSourceTableTask.rst

```diff
@@ -0,0 +1,102 @@
+.. lsst-task-topic:: lsst.pipe.tasks.postprocess.ConsolidateSourceTableTask
+
+##########################
+ConsolidateSourceTableTask
+##########################
+
+``ConsolidateSourceTableTask`` concatenates per-detector Source Tables (dataset `sourceTable`) into one per-visit Source Table (dataset `sourceTable_visit`).
+It only does I/O, and therefore has no run method.
+The inputs have already been transformed to the DPDD-specified columns, and this task assumes that they are sufficiently narrow to fit all tables for a given visit in memory at once.
+
+``ConsolidateSourceTableTask`` is available as a :ref:`command-line task <pipe-tasks-command-line-tasks>`, :command:`consolidateSourceTable.py`.
+
+.. _lsst.pipe.tasks.postprocess.ConsolidateSourceTableTask-summary:
+
+Processing summary
+==================
+
+``ConsolidateSourceTableTask`` reads in all sensor-level Source Tables (dataset `sourceTable`) for a given visit, concatenates them, and writes the result out as a visit-level Source Table (dataset `sourceTable_visit`)
```

Comment:
In some cases in the docs you say "sensor" and in others "detector".  I think we want to standardize on "detector", but I think it should be one or the other.